lib/grape/endpoint.rb

```diff
@@ -319,7 +320,14 @@ def build_helpers
       Module.new { helpers.each { |mod_to_include| include mod_to_include } }
     end
 
-    private :build_stack, :build_helpers
+    def remove_aliased_params
+      return unless route_setting(:aliased_params)
```

Comment:
We were removing the original param key inside the `as` validator, but I removed it (https://github.com/ruby-grape/grape/pull/1852/files#diff-edf496041c46c5c44b88a165b3b32206L11) to place the removal after all validators have run.

Reply:
I don't think `key?` is necessary..

Reply:
I mean `key?` is faster, and more ideomatic, too, because it allows for `nil` value and returns `true` in that case. We should never have a `nil` value assigned here, so that seems "better". Either way it's NBD.

Reply:
I am a little slow :)

Why does removing the original param key _after_ all validators have run fix this issue?

Reply:
Nevermind, I re-read the comment below, I get it now.